tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -637,6 +638,7 @@ bool FindContractionWithBiasAndAddActivation(
 
   // Root of the pattern must be an activation node.
   const auto* node_def = node_view->node();
+  if (node_def == nullptr) return false;
```

Comment:
Did this happen in practice? It's probably a bug somewhere if node def in null.

Reply:
@ezhulenev No, this just pass some tools check.